beetmoverscript/src/beetmoverscript/script.py

```diff
@@ -69,6 +70,25 @@
 log = logging.getLogger(__name__)
 
 
+# push_to_system_addons {{{1
+async def push_to_system_addons(context):
```

Comment:
Side note: we've been moving away from using the god object `context` in other scriptworker-scripts, in favor of passing in more specific arguments to functions. Refactoring beetmoverscript to stop using `context` is definitely massive scope creep, though, so no need to here.

Examples:
- https://github.com/mozilla-releng/scriptworker-scripts/commit/402e25db439a0e26cc047f3711d73d91810b80ca
- https://github.com/mozilla-releng/scriptworker-scripts/commit/3ef1c5c27a6f83eb9d7a7fdf57b9a2fcd0a4a54b

probably others where the commit log doesn't mention `context` :)